src/Form/PostType.php

```diff
@@ -52,15 +52,18 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('title', null, [
                 'attr' => ['autofocus' => true],
                 'label' => 'label.title',
+                'empty_data' => '',
             ])
             ->add('summary', TextareaType::class, [
                 'help' => 'help.post_summary',
                 'label' => 'label.summary',
+                'empty_data' => '',
             ])
-            ->add('content', null, [
+            ->add('content', TextareaType::class, [
```

Comment:
this improve the UX by adding more vertical space to work with the content input